### PR TITLE
Allow for custom error messages

### PR DESCRIPTION
**Pull request by ynnadkrap**
@ankane allows for custom messages to be specified in initializer by overriding hash values. Addresses issue #25

### CODE REVIEW

File: test/strong_migrations_test.rb [hunk 1]
```diff
@@ -242,6 +242,16 @@ def test_version_unsafe
...
 
+  def test_custom_error_message_add_index
+    original_message = StrongMigrations.error_messages[:add_index]
+
+    custom_message = 'This is a custom error message for add_index'
+    StrongMigrations.error_messages[:add_index] = custom_message
+    assert_unsafe AddIndex, custom_message
+
+    StrongMigrations.error_messages[:add_index] = original_message
+  end
```
Comment (ynnadkrap) [inline test/strong_migrations_test.rb]: I'm sure there's a better way to do this..I'm not familiar with minitest.

File: test/strong_migrations_test.rb [hunk 2]
```diff
@@ -242,6 +242,16 @@ def test_version_unsafe
...
 
+  def test_custom_error_message_remove_index
+    original_message = StrongMigrations.error_messages[:remove_column]
+
+    custom_message = 'This is a custom error message for remove_column'
+    StrongMigrations.error_messages[:remove_column] = custom_message
+    assert_unsafe RemoveColumn, custom_message
+
+    StrongMigrations.error_messages[:remove_column] = original_message
+  end
```
Comment (ynnadkrap) [inline test/strong_migrations_test.rb]: I'm sure there's a better way to do this..I'm not familiar with minitest.

**Comment by ankane** [comment]
Hey @ynnadkrap, thanks for the PR 👍  I think custom error messages is outside the scope of what I'd like to support. Feel free to fork and update messages for your own project.

**Comment by ankane** [comment]
Hey @ynnadkrap, decided to add this based on more requests. A similar implementation is now on master. Thanks for your help.

**Pull request closed without merging**